### PR TITLE
Update Royal TSX to v2

### PR DESCRIPTION
**Pull request by jalaziz**

### CODE REVIEW

File: Casks/royal-tsx.rb [hunk 1]
```diff
@@ -1,12 +1,11 @@
 cask :v1 => 'royal-tsx' do
-  version '1.4.6.0'
-  sha256 'e67580fcf344302d81df00a6c43559a9d27dc1cc90428663059a6675d5d09355'
+  version '2.0.0.1000'
+  sha256 '1896d8a12c615ccf6c3b432b425a1ee9ee7c3d74439fc9a87ce2f253ccb775b4'
 
-  url "http://app.royaltsx.com/updates/royaltsx_#{version}.dmg"
-  appcast 'http://app.royaltsx.com/updates_stable.php',
-          :sha256 => 'bb602755a8168a181b849f8d35c543207780d42c6b68ac69715bcdc713e5fd8e'
```
Comment (vitorgalvao) [inline Casks/royal-tsx.rb]: This line should remain.

File: Casks/royal-tsx.rb [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+  url "http://v2.royaltsx.com/updates/royaltsx_#{version}.dmg"
+  appcast 'http://v2.royaltsx.com/updates_beta.php'
   homepage 'http://www.royaltsx.com'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/royal-tsx.rb]: Unless you’re providing a license, please keep the comment there.

File: Casks/royal-tsx.rb (hunk 1 above)
Comment (jalaziz) [inline Casks/royal-tsx.rb]: Wouldn't the SHA be different? I can't seem to find the correct checksum in the feed.

File: Casks/royal-tsx.rb (hunk 2 above)
Comment (jalaziz) [inline Casks/royal-tsx.rb]: I'll change the license to :freemium, based on the information provided here: http://support.royalapplications.com/knowledgebase/articles/488445

Comment (vitorgalvao) [inline Casks/royal-tsx.rb]: Sounds good.

File: Casks/royal-tsx.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/royal-tsx.rb]: It would, yes You’re supposed to check it manually, but you can just leave it as is, since the feature isn’t yet operational.

Comment (jalaziz) [inline Casks/royal-tsx.rb]: I might as well do the right thing. Download the page and shasum it?

Comment (vitorgalvao) [inline Casks/royal-tsx.rb]: > Download the page and shasum it?

Yes. But really, you don’t need to waste time with it. By the time the feature is implemented, the `sha256` you input now will probably be outdated anyway.

**Comment by jalaziz** [comment]
@vitorgalvao I made both changes (for completeness). Let me know if any other changes are required.

**Comment by vitorgalvao** [comment]
Looks good. Thank you.

**Pull request merged**